test/inductor/test_torchinductor.py

```diff
@@ -7772,15 +7809,6 @@ def fn(input_ids) -> torch.Tensor:
         x = torch.randn(2, 2)
         self.common(fn, (x,), atol=0, rtol=0)
 
-    # It's a view so it doens't generate a kernel
-    @expectedFailureCodegenDynamic
-    def test_slice(self):
```

Comment:
Unrelated change, I've moved this next to the other `test_slice` functions and renamed it `test_slice3` as there are already `test_slice` 1 and 2.